site/docs/components/grid-layout/usage.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+### When to use Grid Layout
+
+- Your content is best displayed using a configurable number of columns and rows, such as a [Form](https://www.saltdesignsystem.com/salt/patterns/forms) or a Dashboard.
+
+- For content regions where blocks, or groups, of content are displayed alongside each other, such as the main content area of the application, a Dialog, or a Drawer.
```

Comment:
do we want links for Dialog and Drawer?

Reply:
Yeah, I was going to wait until those were on the site or should we link them to storybook?